README.md

```diff
@@ -1,6 +1,6 @@
-# Opinionated Bioinformatics Project Directory Structure
+# Cookiecutter Bioinformatics project
```

Comment:
I would still keep opinionated in there, because this isn't a widely accepted construction